backend/btrixcloud/orgs.py

```diff
@@ -266,11 +264,23 @@ async def get_max_pages_per_crawl(self, oid: uuid.UUID):
             return org.quotas.maxPagesPerCrawl
         return 0
 
-    async def inc_org_bytes_stored(self, oid: uuid.UUID, size: int):
+    async def inc_org_bytes_stored(self, oid: uuid.UUID, size: int, type_="crawl"):
         """Increase org bytesStored count (pass negative value to subtract)."""
         await self.orgs.find_one_and_update(
             {"_id": oid}, {"$inc": {"bytesStored": size}}
         )
+        if type_ == "crawl":
```

Comment:
very small optimization: could combine it into a single query that's updated to get: `{"$inc": {"bytesStored": size, "bytesStoredCrawls": size}}`